central/networkgraph/flow/datastore/internal/store/rocksdb/flow_impl.go

```diff
@@ -53,6 +54,35 @@ func (s *flowStoreImpl) GetMatchingFlows(ctx context.Context, pred func(*storage
 	return flows, ts, err
 }
 
+func (s *flowStoreImpl) GetFlowsForDeployment(ctx context.Context, deploymentID string) ([]*storage.NetworkFlow, error) {
+	defer metrics.SetRocksDBOperationDurationTime(time.Now(), ops.GetMany, "NetworkFlow")
+	if err := s.db.IncRocksDBInProgressOps(); err != nil {
+		return nil, err
+	}
+	defer s.db.DecRocksDBInProgressOps()
+
+	// Function to match flows referencing the deployment ID passed in.
```

Comment:
@c-du I'm open to suggestions here.  I could use GetMatchingFlows and that would work just fine for Rocks.  But that would be awful for postgres as it would grab all rows and then exclude based on the predicate function.  Since I knew I was just going after deployments I thought it would make sense to do that via query in Postgres.  So I created a new function to keep the interfaces the same.  I could only use that function for postgres and hide it behind the feature flag or I could update the Rocks version to simply call GetMatchingFlows.  

What's the preference?